src/vtk_calculator.py

```diff
@@ -65,31 +70,51 @@ def parse_args():
+    +0.5*exp(-((9*{first}-7)^2+(9*{second}-3)^2)/4)
+    -0.2*exp(-(9*{first}-4)^2-(9*{second}-7)^2)""",
+    "eggholder2d": """-{first}*sin(sqrt(abs({first}-{second}-47)))
+    -({second}+47)*sin(sqrt(abs(0.5*{first}+{second}+47)))""",
+    "rosenbrock2d": "(100*({second}-{first}^2)^2+({first}-1)^2)"
+}
+
 preDefFunctions = {
-    "franke3d": """0.75*exp(((9*x-2)^2+(9*y-2)^2+(9*z-2)^2)/4)+0.75*exp(-(9*x+1)^2/49-(9*y+1)/10-(9*z+1)/10)
-    +0.5*exp(-((9*x-7)^2+(9*y-3)^2+(9*z-5)^2)/4)-0.2*exp(-(9*x-4)^2-(9*y-7)^2-(9*z-5)^2)""",
-    "franke2d(xy)": """0.75*exp(-((9*x-2)^2+(9*y-2)^2)/4)+0.75*exp(-(9*x+1)^2/49-(9*y+1)/10)
-    +0.5*exp(-((9*x-2)^2+(9*y-3)^2)/4)-0.2*exp(-(9*x-4)^2-(9*y-7)^2)""",
-    "franke2d(xz)": """0.75*exp(-((9*x-2)^2+(9*z-2)^2)/4)+0.75*exp(-(9*x+1)^2/49-(9*z+1)/10)
-    +0.5*exp(-((9*x-2)^2+(9*z-3)^2)/4)-0.2*exp(-(9*x-4)^2-(9*z-7)^2)""",
-    "franke2d(yz)": """0.75*exp(-((9*y-2)^2+(9*z-2)^2)/4)+0.75*exp(-(9*y+1)^2/49-(9*z+1)/10)
-    +0.5*exp(-((9*y-2)^2+(9*z-3)^2)/4)-0.2*exp(-(9*y-4)^2-(9*z-7)^2)""",
-    "eggholder2d(xy)": "-x*sin(sqrt(abs(x-y-47)))-(y+47)*sin(sqrt(abs(0.5*x+y+47)))",
-    "eggholder2d(xz)": "-x*sin(sqrt(abs(x-z-47)))-(z+47)*sin(sqrt(abs(0.5*x+z+47)))",
-    "eggholder2d(yz)": "-y*sin(sqrt(abs(y-z-47)))-(z+47)*sin(sqrt(abs(0.5*y+z+47)))",
+    "franke3d": """0.75*exp(-((9*x-2)^2+(9*y-2)^2+(9*z-2)^2)/4)
+    +0.75*exp(-(9*x+1)^2/49-(9*y+1)/10-(9*z+1)/10)
+    +0.5*exp(-((9*x-7)^2+(9*y-3)^2+(9*y-5)^2)/4)
+    -0.2*exp(-(9*x-4)^2-(9*y-7)^2-(9*z-5)^2)""",
     "eggholder3d": """-x*sin(sqrt(abs(x-y-47)))-(y+47)*sin(sqrt(abs(0.5*x+y+47)))
     -y*sin(sqrt(abs(y-z-47)))-(z+47)*sin(sqrt(abs(0.5*y+z+47)))""",
-    "rosenbrock2d(xy)": "(100*(y-x^2)^2+(x-1)^2)",
-    "rosenbrock2d(xz)": "(100*(z-x^2)^2+(x-1)^2)",
-    "rosenbrock2d(yz)": "(100*(z-y^2)^2+(y-1)^2)",
-    "rosenbrock3d": "(100*(y-x^2)^2+(x-1)^2)+(100*(z-y^2)^2+(y-1)^2)"
+    "rosenbrock3d": "(100*(y-x^2)^2+(x-1)^2)+(100*(z-y^2)^2+(y-1)^2)",
+    "franke2d(xy)": twoDFunctions["franke2d"].format(first="x", second="y"),
+    "franke2d(xz)": twoDFunctions["franke2d"].format(first="x", second="z"),
+    "franke2d(yz)": twoDFunctions["franke2d"].format(first="y", second="z"),
+    "eggholder2d(xy)": twoDFunctions["eggholder2d"].format(first="x", second="y"),
+    "eggholder2d(xz)": twoDFunctions["eggholder2d"].format(first="x", second="y"),
+    "eggholder2d(yz)": twoDFunctions["eggholder2d"].format(first="x", second="y"),
+    "rosenbrock2d(xy)": twoDFunctions["rosenbrock2d"].format(first="x", second="y"),
+    "rosenbrock2d(xz)": twoDFunctions["rosenbrock2d"].format(first="x", second="z"),
+    "rosenbrock2d(yz)": twoDFunctions["rosenbrock2d"].format(first="y", second="z"),
 }
 
+def print_predef_functions():
+    print("Available predefined functions are:")
+    for key in list(preDefFunctions.keys()):
+        print(key)
+    print("Please visit the preCICE documentation for more information.")
+    print("https://precice.org/tooling-aste.html")
```

Comment:
We could also print the function definitions. Forwarding to the documentations seems a bit unnecessary for this.

```suggestion
    print("Available predefined functions are:")
    longest = max(map(len, preDefFunctions.keys())
    for name, func in list(preDefFunctions.items()):
        print(f"{name:{longest}} := {func}")
```

Reply:
Function definitions are also moved to here.